backend/tests/routes/Q&A/answers/approve.test.js

```diff
@@ -0,0 +1,49 @@
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../../../../index');
+
+chai.use(chaiHttp);
+
+describe('Approve test for Answers', () => {
+  let qId = '';
+  let aId = '';
+  let approved = '';
+  it('should return questions with status code 200 with question array json object', (done) => {
```

Comment:
Please remove this test as it's already tested under the `approve.test.js`

Reply:
Ma'am we have to test it again to get the `questionId` which is required to get the `answerId` as per the schema of the answer. So we can't remove it.

The only way possible is if we test all the APIs of `question and answers` in one single file